infrastructure/__main__.py

```
@@ -1,5 +1,72 @@
+    create_environment_variable,
+    ALPACA_API_KEY_ID,
+    ALPACA_API_SECRET_KEY,
+    GCP_PROJECT,
+    DATA_BUCKET,
+    DUCKDB_ACCESS_KEY,
+    DUCKDB_SECRET,
+    POLYGON_API_KEY,
+)
+from services import create_service
 import topics  # noqa: F401
 import buckets  # noqa: F401
```

Pylint (3.3.7)

[warning] Unused import buckets

(W0611)